ol/cli/src/server.rs

```diff
@@ -31,7 +31,7 @@ pub async fn start_server(mut node: Node, _run_checks: bool) {
         let interval = interval(Duration::from_secs(10));
         let stream = IntervalStream::new(interval);
         let event_stream = stream.map(move |_| {
-            let vitals = Vitals::read_json(&path);
+            let vitals = Vitals::read_json(&path).unwrap();
```

Comment:
Instead of unwrap() here you want to use `?` like this Vitals::read_json(&path)?

Reply:
That part is actually outdated, currently exiting. So you think is better to propagate it?